pcbdraw/mdrenderer.py

```diff
@@ -43,6 +47,10 @@ def hrule(self):
     def heading(self, text, level, raw=None):
         return '#'*level + " " + text + '\n\n'
 
+    # Old mistune
```

Comment:
Please, explicitly mention which version in the comment. And more preferably, make it only an alias of the `header` method.

Reply:
Ok, added in 8b2158d